qa/evil-tests/build.gradle

```diff
@@ -17,6 +17,7 @@ apply plugin: 'elasticsearch.standalone-test'
 
 dependencies {
   testImplementation "com.google.jimfs:jimfs:1.2"
+  testImplementation "com.google.guava:guava:${versions.jimfs_guava}"
```

Comment:
is this needed? I don't see any changes to these tests?

Reply:
The use of NIO `Path` earlier in some early ES code paths triggers non-default file system providers to be searched, which finds the Jimfs provider, whose static initializer tries to load some Guava classes. I think that this was a pre-existing issue - one cannot load Jimfs without Guava, but was benign if file system providers are not searched.